revolv/project/models.py

```diff
@@ -336,6 +338,17 @@ def amount_donated_organically(self):
         )["amount__sum"] or 0.0
 
     @property
+    def location_street(self):
+        return self.location.split(',')[0]
+
+    @property
+    def location_city_state_zip(self):
```

Comment:
Are there locations in which there might not be a city, or a state, or a zip, or a street? I feel like for street, there probably are. We need to make sure these functions dont error in those cases